src/Parser/DeathKnight/Blood/Modules/Items/Skullflowers_Haemostasis.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS/index';
+import ITEMS from 'common/ITEMS';
+import ItemDamageDone from 'Main/ItemDamageDone';
+import ItemHealingDone from 'Main/ItemHealingDone';
+import Wrapper from 'common/Wrapper';
+
+class SkullflowersHaemostasis extends Analyzer {
+
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasShoulder(ITEMS.SKULLFLOWERS_HAEMOSTASIS.id);
+  }
+
+  buffStack = 0;
+  wastedBuff = 0;
+  maxBuffStacks = 5;
+  damage=0;
+  heal=0;
+  percentBuff=1/6;
+  dsCastWithBuff=false;
```

Comment:
This bool is never used. I assume you meant to use it to set the savedBuffStacks back to 0? Right now it is never set to 0. Since you have to wait for both the healing and damage event to occur before you can set it to 0 I suggest you make 2 bools for this, one for the damage event and one for the healing, and once they're both true set them both to false and savedBuffStacks to 0.